style.css

```diff
@@ -0,0 +1,121 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;600;700;900&display=swap');
+
+* {
+  font-family: Arial, Helvetica, sans-serif;
+  margin: 0;
+}
+
+header {
+  display: flex;
+  background-color: #fff;
+  width: 100%;
+  height: 90px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.mh-logo {
+  display: flex;
+  color: #5969fa;
+  line-height: 20px;
+  text-align: center;
+  margin-left: 24px;
+}
+
+header > .mh-logo {
+  font-size: 18px;
+}
+
+.main-nav {
+  margin-right: 34px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  font-family: Poppins, sans-serif;
+  background-image: url(./Images/bg.png);
+  background-repeat: no-repeat;
+  background-size: 100% 100%;
+  padding: 90px 24px;
+}
```

Comment:
- [ ] Kindly remove the padding for the `left and right (24px)` as you already set a margin of `24px` in the other elements, and according to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) you should have only `24px` margin, adding a padding of `24px` makes it look like `48px` margin. 😉
![image](https://user-images.githubusercontent.com/20567503/177660176-ef6fa2cf-a3bf-47fb-a079-af9f89643b65.png)